pkg/loader/compose/compose.go

```diff
@@ -178,6 +183,50 @@ func loadEnvVars(envars []string) []kobject.EnvVar {
 	return envs
 }
 
+// Convert the Docker Compose v3 volumes to []string (the old way)
+// TODO: Check to see if it's a "bind" or "volume". Ignore for now.
+// See: https://docs.docker.com/compose/compose-file/#long-syntax-2
+func loadV3Volumes(volumes []types.ServiceVolumeConfig) []string {
```

Comment:
I think this is what @surajnarwade is working on in #626 he is trying to improve the data structure of volume at https://github.com/kubernetes-incubator/kompose/pull/626/files#diff-0e2f65ad79c33d7818d9a9312b90391bR111